eyes_selenium/applitools/selenium/visual_grid/dom_snapshot_script.py

```diff
@@ -25,6 +25,8 @@
 MAX_CHUNK_BYTES_GENERIC = 50 * 1024 * 1024
 SCRIPT_POLL_INTERVAL_MS = 1000
 
+CORS_IFRAME_PROCESSING_ENABLED = True
```

Comment:
Probably need clarification about this API. Should it be in Configuration? @danielputerman

Reply:
As far as I know, API is being discussed. I left it here temporarily to aid in stabilisation of SDC suite. It should be replaced with API for sure when it's ready.